tests/api/test_attributes.py

```diff
@@ -93,13 +92,171 @@ def test_attributes_query(user_api_client, product):
             }
         }
     }
-    """
+"""
+
+
+def test_attributes_query(user_api_client, product):
+    attributes = Attribute.objects
+    query = QUERY_ATTRIBUTES
     response = user_api_client.post_graphql(query)
     content = get_graphql_content(response)
     attributes_data = content["data"]["attributes"]["edges"]
```

Comment:
you could add here assert to confirm that `attributes_data` has `len`> 0

Reply:
```suggestion
    attributes_data = content["data"]["attributes"]["edges"]
    assert attributes_data
```

That should be enough 👍